app/views/site/_id.html.erb

```diff
@@ -1,10 +1,20 @@
-<%= javascript_include_tag "edit/id" %>
-
-<div id="map">
-  <% data = { :key => Settings.id_key }
-     data[:lat] = @lat if @lat
-     data[:lon] = @lon if @lon
-     data[:gpx] = trace_data_url(params[:gpx], :format => :xml) if params[:gpx]
-     data[:url] = id_url(:locale => params[:locale]) %>
-  <%= tag.iframe "", :frameBorder => 0, :id => "id-embed", :class => "id-embed", :allowfullscreen => "", :data => data %>
-</div>
+<!DOCTYPE html>
```

Comment:
@geohacker Can you explain more about this change? It seems to take iD out of an iframe and instead embed it directly in the page. This causes CSS conflicts in the icons. But before we try to either undo this commit or fix the CSS conflicts, it would be helpful to know why you made this change, in case it was to address some other problem.